Automattic/wp-calypso
# Add migration instructions events

Related to #91774

## Proposed Changes

* Register events when the user progresses through the steps.

## Why are these changes being made?


* See more context in p9Jlb4-c8w-p2#comment-11779

## Testing Instructions



* Make sure you have the `migration-flow/new-migration-instructions-step` feature flag enabled.
* Create a new site with the Import intent.
* Follow all the steps until the Migration Started step.
* Watch the network requests in your browser.
* Check that an event is registered for every step completed (You can filter `t.gif`).
* Make sure that when you click on a previous step and complete it again, it's not recorded a second time.

## Pre-merge Checklist
